src/test/resources/features/04_employment.feature

```diff
@@ -1729,3 +1729,294 @@ Feature: Employment Journeys - Income Tax Submission
+    When the user clicks the continue button
+    Then the user should be redirected to the "Employment expenses" page
+    When the user clicks the continue button
+    Then the user should be redirected to the "Do you want to claim business travel and overnight expenses?" page
+    When the user selects the no radio button
+    When the user clicks the continue button
+    Then the user should be redirected to the "Do you want to claim for uniforms, work clothes, or tools?" page
+    When the user selects the no radio button
+    When the user clicks the continue button
+    Then the user should be redirected to the "Do you want to claim for professional fees and subscriptions?" page
+    When the user selects the no radio button
+    When the user clicks the continue button
+    Then the user should be redirected to the "Do you want to claim for buying other equipment?" page
+    When the user selects the no radio button
+    When the user clicks the continue button
+    Then the user should be redirected to the "Check your employment expenses" page
+    And the user clicks the save and continue button
+    Then the user should be redirected to the "Employment" page
+
+  Scenario: Individual user as a new submission selecting no to do you want to claim expenses and to skip all expenses pages
+    When the user logs into the service with the following details
+      | Redirect url        | /2021/start |
+      | Credential Strength | strong      |
+      | Confidence Level    | 200         |
+      | Affinity Group      | Individual  |
+      | Nino                | AA133742A   |
+      | Enrolment Key 1     | HMRC-MTD-IT |
+      | Identifier Name 1   | MTDITID     |
+      | Identifier Value 1  | 1234567890  |
+    Then the user should be redirected to the "Update and submit an Income Tax Return" page
+    When the user clicks the continue button
+    Then the user should be redirected to the "Your Income Tax Return" page
+    When the user clicks on the employment link
+    Then the user should be redirected to the "Employment" page
+    And the user clicks on the change link
+    Then the user should be redirected to the "Employment details and benefits" page
+    And the user clicks on the expenses link
+    Then the user should be redirected to the "Check your employment expenses" page
+    When the user clicks on the first change employment expenses link
+    Then the user should be redirected to the "Do you want to claim employment expenses?" page
+    When the user selects the no radio button
+    When the user clicks the continue button
+    Then the user should be redirected to the "Check your employment expenses" page
+    And the user clicks the save and continue button
+    Then the user should be redirected to the "Employment" page
+
+  Scenario: Individual user has prior submission data, updates all expenses fields
+    When the user logs into the service with the following details
```

Comment:
Could also update the radio pages, e.g. after changing the amount, change the question page to no

Reply:
I added a separate test for all No selections being made